<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RaporttiKoostajaWidget</class>
 <widget class="QDialog" name="RaporttiKoostajaWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Kausi</string>
         </property>
         <property name="buddy">
          <cstring>jaksoCombo</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="jaksoCombo"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Kieli</string>
         </property>
         <property name="buddy">
          <cstring>kieliCombo</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KieliCombo" name="kieliCombo"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Vastaanottajat</string>
         </property>
         <property name="buddy">
          <cstring>vastaanottajaEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="vastaanottajaEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Otsikko</string>
         </property>
         <property name="buddy">
          <cstring>otsikkoEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="otsikkoEdit"/>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="textEdit">
     <property name="placeholderText">
      <string>Vapaamuotoinen saateviesti</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="previewButton">
       <property name="text">
        <string>Esikatsele</string>
       </property>
       <property name="icon">
        <iconset resource="../../pic/pic.qrc">
         <normaloff>:/pic/print.png</normaloff>:/pic/print.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="text">
        <string>Lähetä</string>
       </property>
       <property name="icon">
        <iconset resource="../../pic/pic.qrc">
         <normaloff>:/pic/email.png</normaloff>:/pic/email.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KieliCombo</class>
   <extends>QComboBox</extends>
   <header location="global">kieli/kielicombo.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>jaksoCombo</tabstop>
  <tabstop>kieliCombo</tabstop>
  <tabstop>vastaanottajaEdit</tabstop>
  <tabstop>otsikkoEdit</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>previewButton</tabstop>
  <tabstop>sendButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../pic/pic.qrc"/>
 </resources>
 <connections/>
</ui>
